src/Sarif.Converters/TSLintConverter.cs

```diff
@@ -21,41 +20,18 @@ public TSLintConverter()
         
         public override void Convert(Stream input, IResultLogWriter output, OptionallyEmittedData dataToInsert)
         {
-            input = input ?? throw new ArgumentNullException(nameof(input));
-
+            input = input   ?? throw new ArgumentNullException(nameof(input));
```

Comment:
>    [](start = 25, length = 3)

Accidental extra space?

"throw expression" FTW. Nice language feature.

Reply:
was actually intentional but you'd have to be in the code editor to see why i lined things up for readability. we don't have a lot of this sort of formatting as a standard practice (it's pervasive in most C++ code bases I've worked on). reverted.

---
In reply to: [244521769](https://github.com/Microsoft/sarif-sdk/pull/1191#discussion_r244521769) [](ancestors = 244521769)

Reply:
Oh, right, lines up with "output". If I'd noticed that I wouldn't have commented.

---
In reply to: [244605570](https://github.com/Microsoft/sarif-sdk/pull/1191#discussion_r244605570) [](ancestors = 244605570,244521769)